conans/test/integration/toolchains/meson/test_mesontoolchain.py

```diff
@@ -91,7 +91,7 @@ def test_extra_flags_via_conf():
 
     t.run("install . -pr=profile")
     content = t.load(MesonToolchain.native_filename)
-    assert "cpp_args = ['-flag0', '-other=val', '-flag1', '-flag2']" in content
+    assert "cpp_args = ['-flag0', '-other=val', '-flag1', '-flag2', '-D_GLIBCXX_USE_CXX11_ABI=0']" in content
```

Comment:
Here `libcxx` is `None` but `gcc_cxx11_abi = GLIBCXX_USE_CXX11_ABI=0` so it's only added to `cpp_args`